packages/manager/modules/pci/src/projects/new/new.routing.js

```diff
@@ -97,8 +99,30 @@ export default /* @ngInject */ ($stateProvider) => {
           PCI_REDIRECT_URLS,
           `${coreConfig.getRegion()}.paymentMethods`,
         ),
-        newProjectInfo: /* @ngInject */ ($timeout, coreConfig, ovhPaymentMethod,
+        project: /* @ngInject */ ($transition$, PciProjectNewService) => {
+          if ($transition$.params().projectId) {
+            return PciProjectNewService
+              .getProject($transition$.params().projectId);
+          }
+          return null;
```

Comment:
```suggestion
          return $q.when();
```

if `PciProjectNewService.getProject` returns a `Promise`